project.clj

```diff
@@ -0,0 +1,40 @@
+(defproject puppetlabs/clj-shell-utils "1.0.0"
+  :description "Clojure shell execution utilities"
+
+  :min-lein-version "2.9.0"
+
+  :parent-project {:coords [puppetlabs/clj-parent "3.0.0"]
+                   :inherit [:managed-dependencies]}
+
+  :pedantic? :abort
+
+  :test-paths ["test/unit"]
+
+  :plugins [[lein-project-version "0.1.0"]
+            [lein-parent "0.3.1"]]
```

Comment:
I think "0.3.6" is the latest and would be good to use. We just haven't updated existing projects to it yet.